public/app/features/admin/ServerStats.tsx

```diff
@@ -81,14 +85,27 @@ export const ServerStats = () => {
             content={[
               { name: 'Organisations', value: stats?.orgs },
               { name: 'Users total', value: stats?.users },
-              { name: 'Active users in last 30 days', value: stats?.activeUsers },
               ...(config.featureToggles.displayAnonymousStats && stats?.activeDevices
+                ? [{ name: 'Active users in last 30 days', value: activeDevicesAndUsers }]
+                : [{ name: 'Active users in last 30 days', value: stats?.activeUsers }]),
+              ...(config.featureToggles.displayAnonymousStats && stats?.activeDevices && config.anonymousDeviceLimit
                 ? [
-                    { name: 'Active anonymous devices in last 30 days', value: stats?.activeDevices },
-                    { name: 'Active anonymous users in last 30 days', value: Math.floor(stats?.activeDevices / 3) },
+                    {
+                      name: 'Active anonymous users',
+                      value: `${Math.floor(stats?.activeDevices / 3)} / ${Math.floor(config.anonymousDeviceLimit / 3)}`,
+                      highlight: stats?.activeDevices > config.anonymousDeviceLimit,
+                      tabbed: true,
```

Comment:
what does `tabbed` mean in this context? after running this locally, i think it means we add a ` - ` before the stat name to indent it slightly? 

imo `tabbed` suggests actual clickable tabs, maybe just `indent` is better?

Reply:
changed this to be `indent`! 👍 great name